README.md

```diff
@@ -21,3 +21,7 @@ here or reach out on the mailing list.
 
 * Join the architecture sync up, every Tuesday at 3:00 PM UTC, 11:00 AM US/Eastern. Details are in the shared calendar.
 * Recordings are available at [youtube.com/@konflux-ci](https://www.youtube.com/@konflux-ci).
+
+## Special Interest Groups
+
+* [Konflux + Fedora](sigs/fedora.md)
```

Comment:
you reference sigs/fedora.md here, but the file you created is sigs/konflux.md.  Error?

Reply:
Yeah, super error. Will rename file to `sigs/fedora.md`.